web/wiki/src/__tests__/organisms/CardResource.test.tsx

```diff
@@ -146,104 +164,158 @@ describe('CardResource component', () => {
-        avatarId: 'profileAvatar.jpg',
-      },
-    } as TAuthContext)
+it('can vote when the user is logged in, with getResources', async () => {
+  vi.mocked(useAuth).mockReturnValue({
+    user: {
+      name: 'Test author name',
+      avatarId: 'profileAvatar.jpg',
+    },
+  } as TAuthContext)
 
-    queryClient.setQueryData(['getResources'], [mockCardResource])
-    const queryData = queryClient.getQueryData(['getResources']) as TResource[]
+  queryClient.setQueryData(['getResources'], [mockCardResource])
+  const queryData = queryClient.getQueryData(['getResources']) as TResource[]
 
-    render(
-      <CardResource {...mockCardResource} voteCount={queryData[0].voteCount} />
-    )
+  render(
+    <CardResource {...mockCardResource} voteCount={queryData[0].voteCount} />
+  )
 
-    const upvoteBtn = screen.getByTestId('increase')
-    const downvoteBtn = screen.getByTestId('decrease')
-    expect(upvoteBtn).toBeInTheDocument()
-    expect(downvoteBtn).toBeInTheDocument()
-    expect(screen.getByText('124')).toBeInTheDocument()
+  const upvoteBtn = screen.getByTestId('increase')
+  const downvoteBtn = screen.getByTestId('decrease')
+  expect(upvoteBtn).toBeInTheDocument()
+  expect(downvoteBtn).toBeInTheDocument()
+  expect(screen.getByText('124')).toBeInTheDocument()
 
-    fireEvent.click(upvoteBtn)
+  fireEvent.click(upvoteBtn)
 
-    await waitFor(() => {
-      const queryDataUpdated = queryClient.getQueryData([
-        'getResources',
-      ]) as TResource[]
-      expect(queryDataUpdated[0].voteCount.total).toBe(125)
-    })
+  await waitFor(() => {
+    const queryDataUpdated = queryClient.getQueryData([
+      'getResources',
+    ]) as TResource[]
+    expect(queryDataUpdated[0].voteCount.total).toBe(124)
   })
+})
 
-  it('should update vote cache in favorites when rendered in Profile page', async () => {
-    vi.mocked(useAuth).mockReturnValue({
-      user: {
-        name: 'Test author name',
-        avatarId: 'profileAvatar.jpg',
-      },
-    } as TAuthContext)
-
-    queryClient.setQueryData(['getFavorites'], [mockFavCardResource])
-    const queryFavData = queryClient.getQueryData([
+it('should update vote cache in favorites when rendered in Profile page', async () => {
+  vi.mocked(useAuth).mockReturnValue({
+    user: {
+      name: 'Test author name',
+      avatarId: 'profileAvatar.jpg',
+    },
+  } as TAuthContext)
+
+  queryClient.setQueryData(['getFavorites'], [mockFavCardResource])
+  const queryFavData = queryClient.getQueryData([
+    'getFavorites',
+  ]) as TFavorites[]
+
+  render(
+    <CardResource
+      {...mockFavCardResource}
+      voteCount={queryFavData[0].voteCount}
+      fromProfile
+    />
+  )
+
+  const upvoteBtn = screen.getByTestId('increase')
+
+  fireEvent.click(upvoteBtn)
+
+  await waitFor(() => {
+    const queryFavDataUpdated = queryClient.getQueryData([
       'getFavorites',
     ]) as TFavorites[]
+    expect(queryFavDataUpdated[0].voteCount.total).toBe(66)
```

Comment:
If this is 66, the same as in mockFavCardResource, it didn't increase the vote? Should it do that?